### PR TITLE
Added g032 autoencoder files

### PR DESCRIPTION
**Pull request by esleon97**

### CODE REVIEW

**Comment by aobol** [comment]
in this PR you have several files in .ipynb_checkpoints folder, can you remove them? You only need the 3 scripts in CAE_datacleaning folder instead of 8

File: CAE_datacleaning/g032_cae_calib.ipynb [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Define Convolutional Autoencoder\n",
+    "\n",
+    "class waveform_cae_ralph(nn.Module):\n",
+    "    def __init__(self):\n",
+    "        super(waveform_cae_ralph, self).__init__()\n",
+    "        \n",
+    "        self.dropout = nn.Dropout(p=0.1)\n",
```
Comment (aobol) [inline CAE_datacleaning/g032_cae_calib.ipynb]: In the future you will want to have many dropout instances created (i.e. self.do1 = nn.Dropout(0.1), self.do2 = nn.Dropout(0.1)......) instead of using just one for all dropout operations.

File: CAE_datacleaning/g032_cae_phys.ipynb [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Send data to GPU\n",
+    "def preprocess(x):\n",
+    "    return x.to(DEVICE)\n",
+    "\n",
+    "class WrappedDataLoader:\n",
```
Comment (aobol) [inline CAE_datacleaning/g032_cae_phys.ipynb:98]: You don't need to send data to GPU one by one. usually what I do is during training, I send each batch of data to GPU, like this:
for i, (images, labels) in enumerate(train_loader):
            classifier.train()
            images = images.to(DEVICE)
            labels = labels.view(-1,1)
            labels = labels.to(DEVICE)
This might be slightly slower, but it saves the memory on GPU.

**Comment by aobol** [comment]
Hi Esteban, there are still .ipynb_checkpoints file in the BDT folder, could you also remove those?
You should check the "Files changed" tab on the top of your PR, the number of files changed should be only 2.
